apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -161,7 +204,51 @@ class VisualizerModal extends React.Component {
               selectedColumn1={this.state.selectedColumn1}
               selectedColumn2={this.state.selectedColumn2}
             />
+          ) : (
+            <div style={styles.placeholderContainer}>
+              <div style={styles.placeholderText}>
+                Select values to generate a visualization
+              </div>
+              <img src={require('./placeholder.png')} />
+            </div>
           )}
+          <div style={{paddingTop: 20}}>
+            <DropdownField
+              displayName="Filter"
+              options={[1, 2, 3]}
+              disabledOptions={[]}
```

Comment:
Are you setting these options as part of a future PR?

Reply:
Yes, filtering is not implemented yet.